src/Article.jsx

```diff
@@ -29,6 +40,14 @@ function Article() {
           <p className="single-article-body">{article.body}</p>
         </div>
       </div>
+      <section className="comments-section">
+        <h3>Comments</h3>
+        <ul className="comments-list">
+          {comments.map((comment) => (
+            <CommentCard key={comment.comment_id} comment={comment} />
```

Comment:
If there are no comments on an article (i.e. article 37) could we have some sort of message displayed to inform the user?